src/board-explorer-app/board-explorer-app.html

```diff
@@ -1341,6 +1341,8 @@
           this.selected = [ event.model.item.refdes ];
         } else {
           this.selected.splice(index, 1);
+          this.$.boardViewer.setSelected(this.selected);
+          this.updateRoute(this.board, this.selected, null);
```

Comment:
I  removed this couple of lines accidentally in latest commits, adding them again.